Makefile

```diff
@@ -1,6 +1,11 @@
 CC	= gcc
 CFLAGS	= -c -g -O2 -Wall -Wno-pointer-sign
+ARCH	= $(shell /usr/bin/uname -m)
+ifeq ($(ARCH),s390x)
+LDFLAGS	= -rdynamic -lhd -lblkid -lcurl -lreadline -lmediacheck -lqc
+else
 LDFLAGS	= -rdynamic -lhd -lblkid -lcurl -lreadline -lmediacheck
+endif
```

Comment:
Please do something like
```
ifeq ($(ARCH),s390x)
LDFLAGS += -lqc
endif
```
so the arch independent flags can stay in a single separate line.